x-test/end2end/tests/lifecycleDeveloperConsoleApproval.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from '@playwright/test';
+
+test('no javascript errors on supervisor selection page', async ({ page }, testInfo) => {
+  let errors = new Array<Error>;
+  page.on('pageerror', err => {
+    errors.push(err);
+  });
+
+  await page.goto('https://host.docker.internal/Test_Request_Portal/report.php?a=LEAF_start_leaf_dev_console_request');
+  await page.getByRole('button', { name: 'I understand and accept' }).click();
+
+  // Wait for load
+  await expect(page.locator('#xhr')).toContainText('This is a request to access the LEAF Developer Console.');
+
+  await page.getByRole('button', { name: 'Next Next Question' }).click();
+
```

Comment:
This button name is part of the 508 updates and will become 'Next Question'.  There is an image element with a 'next' label causing readers to announce both the label and the text.

I think getByRole('button', { name: 'Next Question' }).last().click() should already work now on the other button - last() since after the update there will be two of them.

Reply:
Updated. Generally we'd want to leverage Playwright's code generator as much as possible.